dashboard/test/ui/features/step_definitions/pd.rb

```diff
@@ -100,11 +158,6 @@
   application = FactoryGirl.create :pd_teacher1819_application, :locked, user: teacher, form_data: application_hash.to_json
   application.update(pd_workshop_id: teachercon.id)
 
-  @users[name] = {
-    email: teacher_email,
-    password: teacher_password
-  }
-
```

Comment:
`generate_user` (called on line 95) already [does this](https://github.com/code-dot-org/code-dot-org/blob/cee0f5a912f6b707dca280237bc5c957d68f20bd/dashboard/test/ui/features/step_definitions/steps.rb#L861-L864).